src/pages/editor.js

```diff
@@ -30,48 +33,45 @@ const getParam = function (val) {
  * This is Sparta
  */
 const initBridge = function (key, sbResolveRelations, setStory) {
+  const { StoryblokBridge } = window;
+
   // Initialize the Storyblok JS Bridge
-  window.storyblok.init({
+  const storyblokInstance = new StoryblokBridge({
+    preventClicks: true,
```

Comment:
The preventClicks option doesn't seem to work for me. I still navigate around in the editor and I am not sure why.

Reply:
I tried to investigate this but no luck so far. Should we open a separate ticket or do you want me to dig some more into this now?

Reply:
Thanks for looking into it. I'll add a ticket to our backlog to follow up with Storyblok.